packages/dds/tree/src/test/utils.ts

```diff
@@ -1046,17 +1047,17 @@ export function rootFromDeltaFieldMap(
 	return rootDelta;
 }
 
-export function createTestUndoRedoStacks(
+export function createClonableUndoRedoStacks(
```

Comment:
I would suggest you revert this rename - it will dramatically reduce the "noise" in this PR. I don't think it's necessary, but if you really did want to rename it, then doing it in a followup PR would be better. That's because it's easy to review a PR where the _only_ changes are a rename, even if many files are touched. It's harder to review a PR where half the changed lines are unimportant renames, but the other half are meaningful code changes.